projects/canopy/src/lib/forms/sort-code/sort-code.notes.ts

```diff
@@ -1,7 +1,41 @@
 export const notes = `
-# Sort Code Component
+# Sort Code Directive
 
 ## Purpose
+Provides a directive to apply formatting and specific validators to the input component.
+
+## Usage
+Import the Sort Code Module into your application:
+
+~~~js
+@NgModule({
+  ...
+  imports: [LgSortCodeModule],
+})
+~~~
+
+and in your HTML:
+
+~~~html
+<lg-input-field>
+  Sort Code
+  <lg-hint *ngIf="hint">Must be 6 digits long</lg-hint>
+  <input lgInput lgSortCode formControlName="sortCode" />
+</lg-input-field>
+~~~
+
+**Note: both the lgInput and lgSortCode directives have to be present on the input element.**
+
+## Validators
+The sort code directive automatically adds the \`required\` and \`pattern\` validators by default.
```

Comment:
It would be great to expand on what format is allowed by the \`pattern\` validator, and also include a note about the auto-formatting of the input on blur.